contrib/image-preview/nbrun

```diff
@@ -11,5 +11,5 @@ mkfifo "$FIFO_UEBERZUG"
 trap cleanup EXIT
 tail --follow "$FIFO_UEBERZUG" | ueberzug layer --silent --parser bash &
 
-newsboat
+newsboat $@
```

Comment:
It might be useful to put the `$@` within double quotes:
```suggestion
newsboat "$@"
```

See for some discussion:
- https://stackoverflow.com/questions/3811345/how-to-pass-all-arguments-passed-to-my-bash-script-to-a-function-of-mine/3816747#3816747
- https://stackoverflow.com/questions/4824590/propagate-all-arguments-in-a-bash-shell-script/4824637#4824637

Other than that, looks like a nice/useful change.